### ISSUE TITLE
[BUG]: Black Screen, no data view

### ISSUE THREAD

**Issue by Finkman** [AykutSarac/jsoncrack-vscode#11]
### Issue description

I installed `jsoncrack` v1.0.8 on VS-Code v1.73.1.
When I open a json file and then start `jsoncrack`, it just displays an empty black window.

![grafik](https://user-images.githubusercontent.com/12600527/203491756-fdee9414-d8cd-4ecc-a49e-729bbac1108a.png)

After a restart of code, each start of `jsoncrack` pops up a message:

![grafik](https://user-images.githubusercontent.com/12600527/203491968-0d1d9fe6-3dd3-4493-943d-3e8f936ccc48.png)

As I said, this message only comes up, after vs code has restarted and  `jsoncrack` is started again.

### Operating system

- OS: Ubuntu 22.04
- VS-Code v1.73.1
- `jsoncrack` v1.0.8

### Priority this issue should have

High (immediate attention needed)

**Comment by Kugeleis** [comment]
same here VS code latest windows 10 and Linux Ubuntu 22.04

**Comment by spkane** [comment]
This is a duplicate of: https://github.com/AykutSarac/jsoncrack-vscode/issues/10

**Comment by mforkel** [comment]
Same here: VSCode 1.76.0, Windows 10, WSL 1.1.3.0

**Comment by AykutSarac** [comment]
Merging under #29

**Comment by mforkel** [comment]
After #29 was closed, I upgraded to version 2.0.2 of the extension. But I still get nothing but a light grid on a gray background (VS Code 1.81.1, Node.js 16.17.1, WSL 1.2.5.0).

**Comment by AykutSarac** [comment]
Can you do the following steps?

1. Press `F1`
3. to the input: `> Developer: Toggle Developer Tools`
4. Run JSON Crack on a JSON file
5. Screenshot the console

Additionally it'd be helpful if you'd record the whole and share it here.

**Comment by mforkel** [comment]
Thanks for your reply and sorry that my report probably was a false alarm. This time, I got a very nice looking graph from "Entable JSON Crack visualization". I'm not sure what went wrong the first time. May be, reloading after the upgrade to 2.0.2 wasn't enough? For this test, I opened a new VS Code window. 

I found two entries in the console that seem to be related to your extension:
```
DevTools failed to load source map: Could not load content for https://vscode-remote+wsl-002bdebianbullseye.vscode-resource.vscode-cdn.net/home/malte/.vscode-server/extensions/aykutsarac.jsoncrack-vscode-2.0.2/build/static/css/main.bd354a94.css.map: Connection error: net::ERR_NAME_NOT_RESOLVED
DevTools failed to load source map: Could not load content for https://vscode-remote+wsl-002bdebianbullseye.vscode-resource.vscode-cdn.net/home/malte/.vscode-server/extensions/aykutsarac.jsoncrack-vscode-2.0.2/build/static/js/main.c0312bba.js.map: Connection error: net::ERR_NAME_NOT_RESOLVED
```
Please let me know if more information would be helpful. I have taken a screenshot of the developer tools' console, but I not sure what you mean by "record the whole".

**Comment by AykutSarac** [comment]
Thanks for sharing and glad it sorted out!
